### DISCUSSION TITLE
Rshiny and MFA

### DISCUSSION THREAD

**Discussion by JHObsidian** [community/community#36594] [New to GitHub]
I have a shiny app that I'd like to deploy on shinyapps.io, however I need to use MFA since there is PII in the data. You can set up the authentication through GitHub and was hoping I could leverage GitHub's MFA functionality to require users to enable MFA. To do this would I think I need to create accounts for my users and then manage them myself? Would this even work and is there a better way to do this?

Thank you!!

**Comment by ossanna16** [comment]
Hey there @JHObsidian and thank you for asking a good question! I'm glad you found our community 🙂
